news/2022-08-15-15-anniversary-art-contest.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "15th Anniversary Art Contest"
+date: 2022-08-15 12:00:00 +0000
+---
+
+With osu!'s 15th anniversary approaching this September, it's time to take a look back at the past from the future with our latest art contest!
+
+![](https://assets.ppy.sh/contests/153/header.jpg)
+
+On a balmy spring evening (the 16th, to be exact) during September back in 2007, some dude by the name of Dean "peppy" Herbert released the first build of a niche rhythm game designed to emulate the gameplay of the recent cult hit *osu! Tatakae! Ouendan!* to the public.
+
+The rest, is quite literally... history. A history that includes hundreds of influential creators and contributors and thousands of adoring and supportive players that have taken osu! from a pet project run by one guy on forum software that is possibly older than your parents to the sprawling and eclectic behemoth that it has become.
+
+And so, as the celebration of osu!'s 15th year draws ever closer, what better way to get the festivities going than with a fan art contest of historical proportions? Thus, we present this contest's theme:
+
+**Draw any one or combination of the official osu! mascots (or community mascots) exploring any part of osu!'s history!**
+
+Returning to the wide themes that made the A Place To Belong contest so interesting, we're foraying this time into something just as wide but a little more concrete. If you're a long-time player, you might already be familiar with a lot of the historical particulars, but in the event you're not, [check out the osu!wiki](/wiki/History_of_osu!) for some pointers on where to start. You may also want to consult any one of the many videos put out by content creators on the topic too.
+
+Speaking of pointers, here's a few suggestions to get the juices flowing:
+
+- The mascots enjoying iconic representations of popular maps from each game mode
+- Time-travelling mascots crying in despair after they realize they've travelled to 2016 and can no longer play any beatmaps properly
```

Comment:
I know this is an idea, but I'm unsure what you mean by this idea and how 2016 is a major point in beatmaps.

Reply:
it's a joke about newer players having considerable difficulty engaging with older content in some cases (usually regarding low approach rate reading)

Reply:
I believe that within the community 2016 is widely seen as the year mapping drastically changed and the likes of Monstrata, Sotarks etc., got really popular